scss/typography/_helpers.scss

```diff
@@ -38,6 +38,39 @@ $subheader-margin-bottom: 0.5rem !default;
 /// @type Number
 $stat-font-size: 2.5rem !default;
 
+////
+/// @group typography-helpers-dropcaps
+////
+
+/// Default font size for dropcaps 
+/// @type Number
+$dropcaps-font-size: 5.20em !default;
+
+/// Default line height for dropcaps
+/// @type Number
+$dropcaps-line-height: 0.5 !default;
+
+/// Default vertically aligned position for dropcaps
+/// @type String
+$dropcaps-vertical-align: baseline !default;
+
+/// Default text transform for dropcaps
+/// @type String
+$dropcaps-text-transform: uppercase !default;
+
+/// Deafult margin for dropcaps
+/// @type Numbers
+$dropcaps-margin: 0.192em 0.0962em 0 0 !default;
+
+@mixin foundation-dropcaps() { 
```

Comment:
`foundation-*` mixins are only used to generate default styles. For now, component mixins does not have a prefix (but I agree they should have one).
So please use just `dropcaps`.